models/mento/celo/mento_celo_pools_v1.sql

```diff
@@ -0,0 +1,29 @@
+{{
+  config(
+    schema = 'mento_celo',
+    alias = 'pools_v1',
+    materialized = 'table'
+  )
+}}
+
+select
+  'celo' as blockchain,
+  'mento' as project,
+  'v1' as version,
```

Comment:
can you do me a favor and double check the usage of `version` in other dex pool models? do they preface with `v` in front of the version number?

Reply:
yeah, we can fix in later PR for others, no worries